lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -378,6 +410,32 @@ private function getPropertyImports(ReflectionProperty $property)
         return array_merge($classImports, $traitImports);
     }
 
+    /**
+     * Retrieves imports for constants.
+     *
+     * @param \ReflectionClassConstant $constant
```

Comment:
This one can be removed

Reply:
Depends on coding style. These useless annotations are at all methods in this file. Should I remove them all?

Reply:
Please remove this one, you can remove the others if you want to, as a bonus, in a separate commit

Reply:
Removed.